src/reducers/url-state.js

```diff
@@ -193,7 +202,7 @@ function hidePlatformDetails(state: boolean = false, action: Action) {
   }
 }
 
-function threadOrder(state: ThreadIndex[] = [], action: Action) {
+function threadOrder(state: ThreadIndex[] = EMPTY_THREAD_LIST, action: Action) {
```

Comment:
useless because it's the initial state + we change it quickly when we receive the profile.